cypress/integration/e2e/header-control-menu/menu-plik.spec.ts

```diff
@@ -16,7 +16,7 @@ context("Menu 'plik'", () => {
 
   describe("Load schedule", () => {
     it("Should be able to load and show the schedule", () => {
-      cy.loadSchedule();
+      cy.loadScheduleToMonth();
```

Comment:
Zmieniłem nazwę komendy, ponieważ dodałem do niej parametry określające na jaki miesiąc chcemy wgrywać grafik. Domyślnie jest to 11.2020.